src/components/container/RestingHeartRateCalendar/RestingHeartRateCalendar.stories.tsx

```diff
@@ -38,4 +38,20 @@ SampleData.args = {
   year: 2022,
   month: 3,
   showPreviewData: "WithData",
-};
\ No newline at end of file
+};
+
+export const Live = Template.bind({});
```

Comment:
@brandoncharnesky I recognize these items had already existed. Hoping they can be cleaned up as part of this PR. The Default and Sample Data stories do not align with the other components. 
I am advocating for
1) The current Default story be renamed to NoData
2) The Sample Data to be renamed to Default,

Reply:
Good catch, I have renamed the stories.  The first story is now Default, which loads the sample data.  The NoData case is now named NoData.